snykTask/src/task-args.ts

```diff
@@ -57,8 +62,12 @@ class TaskArgs {
       }
     }
   }
+
+  // disallow snyk code monitor which follows --report workflow
   public shouldRunMonitor(snykTestSuccess: boolean): boolean {
-    if (this.monitorWhen === 'always') {
+    if (this.testType == TestType.CODE) {
```

Comment:
question: Isn't this redundant, as monitorWhen is taking care of code not running monitor?